toc.md

```diff
@@ -129,7 +129,7 @@ docs.vmware.com is built.
                 - [Install the Tanzu Insight CLI plug-in](cli-plugins/insight/cli-installation.md)
                 - [Configure the Tanzu Insight CLI plug-in](cli-plugins/insight/cli-configuration.md)
                 - [Configure target endpoint and certificate](scst-store/using-encryption-and-connection.md)
-                - [Configure access tokens](scst-store/create-service-account-access-token.md)
+                - [Configure access tokens](scst-store/configure-access-token.md)
```

Comment:
Now that `cli-configuration.hbs.md` contains endpoint and access token guides, you can remove these links to the store docs. This will help fix our navigation as well.